db/db_local/backup.go

```diff
@@ -202,5 +284,65 @@ func loadBackup(ctx context.Context) error {
 		return err
 	}
 
-	return os.Remove(getBackupLocation())
+	return nil
+}
+
+// splitTocMatViewRefreshData writes back the TableOfContents into a two temporary files:
+// 	1. without REFRESH MATERIALIZED VIEWS
+//  2. only REFRESH MATERIALIZED VIEWS
+//
+// This needs to be done because REFRESHING cannot work in restore, since the BackUP will always set
+// a blank search path before commencing and the MATERIALIZED VIEWS may have functions with
+// unqualified table names
+func splitTocMatViewRefreshData(ctx context.Context, toc []string) (string, string, error) {
```

Comment:
awkwards function name - can we improve

Reply:
also inconsistent casing with `getTOCFromBackup(`

Reply:
renamed both these functins